src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupOperation/BackupOperation.cs

```diff
@@ -229,6 +231,12 @@ public override void Execute()
                                 this.backup.Devices.AddDevice(destName, DeviceType.File);
                             }
                             break;
+                        case (int)DeviceType.Url:
+                            if (this.backupDeviceType == BackupDeviceType.Url)
```

Comment:
What's the difference between the device type and the backupDeviceType? And shouldn't we be throwing or something if the backupDeviceType isn't URL here? Or is that an expected scenario?

Reply:
Hm, ok. This seems weird but since the code was already like this I guess it's fine to leave as is for now...would be nice to see if you can figure out why these checks would be necessary as you're working on this code though.